CHANGELOG-1.x.md

```diff
@@ -1,3 +1,5 @@
+- [](https://github.com/patternfly/patternfly-elements/commit/) fix: Updating PFElement.log (aka this.log) so it logs arrays and objects, instead of converting them to strings
```

Comment:
```suggestion
# 1.6.1 (2021)

- [](https://github.com/patternfly/patternfly-elements/commit/) fix: Updating PFElement.log (aka this.log) so it logs arrays and objects, instead of converting them to strings
```

Reply:
Adding the version heading to the top